man/fi_usnic.7.md

```diff
@@ -283,6 +283,24 @@ is the destination host is locally connected, and `1` otherwise.
 
 See fi_ext_usnic.h for more details.
 
+# VERSION DIFFERENCES
+
+## New naming convention for fabric/domain starting with libfabric v1.4
+
+The release of libfabric v1.4 introduced a new naming convention for fabric and domain. However the usNIC provider
+remains backward compatible with applications supporting the old scheme.
```

Comment:
Add "and decides which one to use based on the version passed to `fi_getinfo`" here.